pkg/reconciler/kubernetes/internal/warnings.go

```diff
@@ -1,4 +1,4 @@
-package kubeclient
+package internal
```

Comment:
Why not to move those things to package `kubeclient` inside of `internal`?

Reply:
Would also be fine, I just renamed the `kubeclient` package as is and did not see the need for a single sub-package.
Package `kubernetes` defines the `API` and internal just has the implementation.